public/src/ux.js

```diff
@@ -369,6 +369,7 @@ $('#bitconverterprompt').append(`
  */
 $('#bitconverter').click(() => {
     $('#bitconverterprompt').dialog({
+        resizable:false,
```

Comment:
Why resizable false, it is a good thing right?

Reply:
I disabled resizable for all dialogs, resizing dialogs seemed to break to inner html making it look bit odd,... anyways disablling it doesn't seem to have any negative results on the usability